Configuration/PyReleaseValidation/python/relval_nano.py

```diff
@@ -0,0 +1,364 @@
+from  Configuration.PyReleaseValidation.relval_steps import *
+import math
+
+workflows = Matrix()
+
+class WFN:
+    def __init__(self):
+        self.index=0
+        self.subindex=0
+    def __call__(self):
+        offset=2500
+        if self.subindex==100:
+            print("this is not going to work nicely")
+            self.subindex=0/0
+        r=float(f'{offset}.{self.index}{self.subindex:02d}')
+        self.subindex+=1
+        return r
+    def next(self):
+        self.index+=1
+        self.subindex=0
+    def subnext(self):
+        # go to the next tenth for the subindex 10 because of 02d formating
+        self.subindex = math.ceil(self.subindex/10.)*10
+
+_runOnly20events={'-n':'20'}
+_run10kevents={'-n':'10000'}
+
+steps['NANO_data'] = merge([{'-s':'NANO,DQM:@nanoAODDQM',
+                             '--process':'NANO',
+                             '--data':'',
+                             '--eventcontent':'NANOAOD,DQM',
+                             '--datatier':'NANOAOD,DQMIO',
+                             '-n':'10000',
+                             '--customise_command':'process.MessageLogger.cerr.FwkReport.reportEvery=100;process.Timing=cms.Service("Timing",summaryOnly=cms.untracked.bool(True));process.SimpleMemoryCheck=cms.Service("SimpleMemoryCheck",ignoreTotal=cms.untracked.int32(1))',
```

Comment:
Hi @vlimant 

Two suggestions here:

1. You may need `"` in `'process.Message.....'`, i.e. `:'"process.MessageLogger....."'`

2. I think you it will not work with `process.Timing` and `process.SimpleMemoryCheck` in customise_command. Maybe you can use
```
'--customise':'"Configuration/DataProcessing/Utils.addMonitoring"',
                             '--customise_command':'"process.MessageLogger.cerr.FwkReport.reportEvery=100"',
```

Reply:
thanks indeed. I think we only need the `addMonitoring` part. @smuzaffar is that something added by default during PR test?